### ISSUE TITLE
Not listing all the environments

### ISSUE THREAD

**Issue by anaclg** [iann0036/cloud9-sync#13]
Hi,

I was able to setup Live Sync, but only the first 25 environments from the company account were listed. Is there a limitation? When I try to add my specific environment by name it is not found. Is there something I am missing? Is there also a way to list only my environments instead of the hole company account?

Thanks in advance.

**Comment by iann0036** [comment]
Hi Ana,

I've just pushed 0.9.8 out which should paginate past 25 environments. I hope this doesn't break your listings, so let me know if that works out for you.

I'll also see if I can put in a setting to filter by environment owner.

**Comment by anaclg** [comment]
Hi Ian,

I still can´t see more than 25 environments. I updated the extension to version 0.9.8, gave a reset on the settings and nothing. Is there anything else I could try?

**Comment by iann0036** [comment]
Hi Ana,

Looks like I missed a step when implementing the pagination code 🤦‍♂️. I've released 0.9.9 which should hopefully fix that.

I've also included a setting to filter out based on the environment owner, so you should be able to open the settings and add your username to filter out the others in the listing.

Let me know how it goes.

**Comment by jbruett** [comment]
This is still an issue in 11.3
```
{
    "__type": "ValidationException",
    "message": "1 validation error detected: Value '[list of environments]' at 'environmentIds' failed to satisfy constraint: Member must have length less than or equal to 25"
}
```

**Comment by iann0036** [comment]
Thanks for flagging that issue @jbruett.

I've pushed a fix for that as 0.11.4, but I can't fully test. Could you give it a shot?

**Comment by jbruett** [comment]
I'm now able to enumberate the appropriate environments based on the access account provided.
